.github/workflows/test_and_update.yml

```diff
@@ -3,7 +3,6 @@ on:
   workflow_dispatch:
   release:
     types: [created]
-  pull_request:
```

Comment:
Yossi, agree on this change - this will mean we only run on the main branches?

Reply:
Will have to change the requirements for tesing though as `build` is required to pass

Reply:
yes exactly. And we test the rucio frontend then https://github.com/XENONnT/base_environment/pull/1109/commits/e64a5230da02b8a4ed20e39198cd037e77e9581b